### ISSUE TITLE
[QUESTION] : libwebp vulnerabilty

### ISSUE THREAD

**Issue by Wojtaz0w** [SimpleMobileTools/Simple-Gallery#2990]
Is Gallery Pro affected by CVE-2023-4863? Since it supports opening WebP them it would make sense. Also if it fixed in latest update let us know in changelog as this bug is critical.

**Comment by licaon-kter** [comment]
Update to https://github.com/zjupure/GlideWebpDecoder/releases/tag/2.4 or better yet https://github.com/zjupure/GlideWebpDecoder/releases/tag/2.6 I guess...

I'm updating the recipe for the kotlin changes and I'd like to flag older versions with `KnownVuln` but also have a fixed update for users to update to.

**Comment by licaon-kter** [comment]
Now I can just 
```
      - sed -i -e 's/zjupureWebpdecoder = "2.3.4.15.1"/zjupureWebpdecoder = "2.6.4.16.0"/'
        ../gradle/libs.versions.toml
```
but I'll need your @tibbi okay for this, before you get a chance to release a version with the fixed lib

**Comment by licaon-kter** [comment]
Also, fyi since you use this lib for animated webp, https://github.com/penfeizhou/APNG4Android/issues/207

**Comment by licaon-kter** [comment]
Updated without modifying anything: https://gitlab.com/fdroid/fdroiddata/-/commit/1c0819793bc88082f3a1916c637ed5b493da2764

Ping us when next version is ready if it fixes stuff

**Comment by licaon-kter** [comment]
fyi

**Comment by tibbi** [comment]
sure, but has been fixed in 6.28.1 already
